weed/server/filer_server_handlers.go

```diff
@@ -220,5 +223,10 @@ func (fs *FilerServer) maybeCheckJwtAuthorization(r *http.Request, isWrite bool)
 
 func (fs *FilerServer) filerHealthzHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Server", "SeaweedFS Filer "+util.VERSION)
-	w.WriteHeader(http.StatusOK)
+	if _, err := fs.filer.Store.FindEntry(context.Background(), filer.TopicsDir); err != nil && err != filer_pb.ErrNotFound {
```

Comment:
For a fresh installation, the topics directory may not exist.

Reply:
Therefore, we skip the ErrNotFound error